app/assets/javascripts/discourse/tests/acceptance/bookmarks-test.js

```diff
@@ -26,6 +27,16 @@ acceptance("Bookmarking", function (needs) {
 
   needs.hooks.beforeEach(() => (steps = []));
 
+  const topicResponse = Object.assign({}, topicFixtures["/t/280/1.json"]);
```

Comment:
No idea if this is the nicest way to do this, I just needed extra HTML in the OP to test this.

Reply:
I don't get it, why do you need a new object?
`const topicResponse =  topicFixtures["/t/280/1.json"]` is not working?

Reply:
@lis2 hmm good question, I can probably get rid of this!